test/functional/test_framework/p2p.py

```diff
@@ -188,6 +188,10 @@ def peer_connect_helper(self, dstaddr, dstport, net, timeout_factor):
         self.on_connection_send_msg = None
         self.recvbuf = b""
         self.magic_bytes = MAGIC_BYTES[net]
+        if dstport != 0:
+            self.p2p_connected_to_node = True
+        else:
+            self.p2p_connected_to_node = False
```

Comment:
nit: you can assign the comparison directly

Reply:
Done